introduction.md

```diff
@@ -1,11 +1,19 @@
 # Introduction
 
-Public code is both civil code (like policy or regulation) and computer source code (such as software and algorithms) executed in a public context, by humans or machines.
+The Standard for Public Code is a set of criteria that supports public organisations in developing and maintaining software together. 
```

Comment:
This needs a mention of policy as well (especially since you start with that the next sentence)